pkgs/development/r-modules/default.nix

```
@@ -571,6 +572,7 @@ let
     rsvg = [ pkgs.librsvg.dev ];
     ssh = with pkgs; [ libssh ];
     s2 = [ pkgs.openssl.dev ];
+    opencv = [ pkgs.opencv pkgs.gtk2-x11.dev ];
```

```suggestion
    opencv = [ pkgs.opencv ];
```
gtk2 didn't help, so let's remove it. I think we can live with this error for now. I tried a few things, but didn't help either.